Assets/MixedRealityToolkit.Tests/PlayModeTests/PressableButtonTests.cs

```diff
@@ -264,19 +333,23 @@ public IEnumerator ScaleWorldDistances([ValueSource(nameof(PressableButtonsTestP
             float pressDistance = button.PressDistance;
             float releaseDistance = pressDistance - button.ReleaseDistanceDelta;
 
-            Vector3 startPushDistanceWorld = button.GetWorldPositionAlongPushDirection(startPushDistance);
-            Vector3 maxPushDistanceWorld = button.GetWorldPositionAlongPushDirection(maxPushDistance);
-            Vector3 pressDistanceWorld = button.GetWorldPositionAlongPushDirection(pressDistance);
-            Vector3 releaseDistanceWorld = button.GetWorldPositionAlongPushDirection(releaseDistance);
+            Vector3 zeroPushDistanceWorld = button.GetWorldPositionAlongPushDirection(0.0f);
```

Comment:
New in this iteration, I get the zero point in ScaleWorldDistances in addition to ScaleLocalDistances.